lib/src/main/java/com/orange/ods/compose/component/textfield/password/OdsPasswordTextField.kt

```diff
@@ -88,7 +88,10 @@ fun OdsPasswordTextField(
     characterCounter: (@Composable () -> Unit)? = null
 ) {
     val odsPasswordTextFieldState = rememberOdsPasswordTextFieldState()
-
+    if (!visualisationIcon) {
+        odsPasswordTextFieldState.passwordVisible.value = false
+    }
+    
     Column {
```

Comment:
I found another bug, when the password text field is disabled, the eye icon is always clickable... :/